src/core/operations/DateTime.js

```diff
@@ -1,3 +1,5 @@
+import Decimal from "../lib/decimal.min.js";
```

Comment:
Libraries should be imported through npm if at all possible (`npm install --save decimal.js`), rather than being included statically. However in this case, I would recommend using [`jsbn`](https://www.npmjs.com/package/jsbn) instead as it is already included in this project (see [`IP.js`](https://github.com/gchq/CyberChef/blob/master/src/core/operations/IP.js#L3)) and provides similar functionality. The [Design Principals](https://github.com/gchq/CyberChef/wiki/Contributing#design-principals) on the project wiki go into this reasoning in a bit more detail.